### PR TITLE
Add types for bulma-calendar

### PR DESCRIPTION
**Pull request by remcohaszing**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/bulma-calendar/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+
+    /**
+     * List of disabled dates
+     */
+    disabledDates?: any[];
+
+    /**
+     * List of disabled week days
+     */
+    disabledWeekDays?: any[];
```
Comment (peterblazejewicz) [inline types/bulma-calendar/index.d.ts]: this looks it should support `string` as well:
https://github.com/Wikiki/bulma-calendar/blob/297506683763e7d5d3becaceec8c3d866c39faa5/src/js/datePicker/index.js#L494

File: types/bulma-calendar/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,472 @@
+// Type definitions for bulma-calendar 6.0
+// Project: https://creativebulma.net/components/calendar
```
Comment (peterblazejewicz) [inline types/bulma-calendar/index.d.ts]: Can this be replaced with:
https://github.com/Wikiki/bulma-calendar
? The url from `package.json` seems to be dead (404)

Comment (remcohaszing) [reply types/bulma-calendar/index.d.ts]: I used https://creativebulma.net/product/calendar/demo, because this contains the documentation.

Comment (peterblazejewicz) [reply types/bulma-calendar/index.d.ts]: that's ok. Can we swithc to that one? The url from package seems to be dead one.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@remcohaszing thx for the contribution!

**Pull request merged**